app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsCardView.kt

```diff
@@ -0,0 +1,147 @@
+package org.wikipedia.feed.suggestededits
+
+
+import android.content.Context
+import android.net.Uri
+import android.text.TextUtils
+import android.view.View
+import android.widget.FrameLayout
+import io.reactivex.annotations.NonNull
+import io.reactivex.disposables.CompositeDisposable
+import kotlinx.android.synthetic.main.fragment_add_title_descriptions_item.view.*
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.dataclient.restbase.page.RbPageSummary
+import org.wikipedia.feed.view.DefaultFeedCardView
+import org.wikipedia.feed.view.FeedAdapter
+import org.wikipedia.page.PageTitle
+import org.wikipedia.util.ResourceUtil
+import org.wikipedia.util.StringUtil
+import org.wikipedia.views.ItemTouchHelperSwipeAdapter
+
+class SuggestedEditsCardView(context: Context) : DefaultFeedCardView<SuggestedEditsCard>(context), ItemTouchHelperSwipeAdapter.SwipeableView {
+    interface Callback {
+        fun onSuggestedEditsCardClick(@NonNull pageTitle: PageTitle, @NonNull sourceDescription: String, @NonNull sourceLangCode: String, @NonNull view: SuggestedEditsCardView)
+    }
+
+    private val disposables = CompositeDisposable()
+    var isTranslation: Boolean = false
+    private var sourceDescription: String = ""
+    private val app = WikipediaApp.getInstance()
+    private var summary: RbPageSummary? = null
+    private var sourceLangCode: String = app.language().appLanguageCode
+    private var targetLangCode: String = app.language().appLanguageCodes.get(1)
+    var targetPageTitle: PageTitle? = null
+
+    init {
+        View.inflate(getContext(), R.layout.fragment_add_title_descriptions_item, this)
+    }
+
+    override fun setCard(@NonNull card: SuggestedEditsCard) {
+        super.setCard(card)
+        sourceLangCode = app.language().appLanguageCode
+        targetLangCode = app.language().appLanguageCodes.get(1)
```

Comment:
Not sure why to set them up again

Reply:
The RTL bug was for the same reason... the view would only update the card ... so the fields had to be updated... i can remove initialization if it makes more readable..

Reply:
Yes, I think they should not be set up twice.

Reply:
Done